workflow/tests/hpxml_translator_test.rb

```diff
@@ -642,10 +646,22 @@ def _verify_outputs(runner, rundir, hpxml_path, results)
 
     # Enclosure Walls/RimJoists/FoundationWalls
     (hpxml.walls + hpxml.rim_joists + hpxml.foundation_walls).each do |wall|
-      next unless wall.is_exterior
-
       wall_id = wall.id.upcase
 
+      # Common walls
+      next unless wall.exterior_adjacent_to == wall.interior_adjacent_to
```

Comment:
Ah, you're right. I moved this test on the top of this loop because of `next unless wall.is_exterior` in line 663. I guess this will need to be replaced with an IF statement.

Reply:
Sounds right.